### ISSUE TITLE
Add a "Have comments on this article"  Link at the bottom of the full article view footer

### ISSUE THREAD

**Issue by roman87** [TeamMentor/TM_4_0_Design#713]
In the same manner as we have "Found a bug send us an email" we should add a similar link at the bottom of the full article view. The action should be exactly the same - a mailto support

**Comment by luchia** [comment]
This seems like a double up for no reason. With this added link, we now have three links to email the same email - once in the navigation, once in the article and once in the copyright. Unless we want to remove the email from the copyright and move it to the article instead?

**Comment by DinisCruz** [comment]
I think this is REALY important, in fact I would say that more than a link we should have a FORM to send a message directly to us (why make the user have to open up the email and add the link to the current page, when we can just have a nice form for him at the bottom of the page?).

This is specially important in the current beta stage, since part of the exercise is to improve content. We need to make it REALLY EASY to give us feedback. 

I was going to say that in flare we can hide it better, but thinking about it, @luchia the css trick you tried on the right-hand-side navigation for the filters, would actually work really well here (since the link and movement would happen at the bottom of the page)

Summing up:
 - email link with pre populated to and subject : ASAP (1x effort)
 - form to fill in: after that (3x effort)

**Comment by michaelhidalgo** [comment]
I also believe this is very important. I know it is not 100% related to
this discussion, but I believe we also need to support the fact that a
client would like to have articles for any underlaying technology.

It would be great to have a kind of feature that allows them to "Request an
article" or "Request content" and have a kind of categories.


2015-04-25 9:19 GMT-06:00 Dinis Cruz <notifications@github.com>:

> I think this is REALY important, in fact I would say that more than a link
> we should have a FORM to send a message directly to us (why make the user
> have to open up the email and add the link to the current page, when we can
> just have a nice form for him at the bottom of the page?).
>
> This is specially important in the current beta stage, since part of the
> exercise is to improve content. We need to make it REALLY EASY to give us
> feedback.
>
> I was going to say that in flare we can hide it better, but thinking about
> it, @luchia <https://github.com/luchia> the css trick you tried on the
> right-hand-side navigation for the filters, would actually work really well
> here (since the link and movement would happen at the bottom of the page)
>
> Summing up:
>
>    - email link with pre populated to and subject : ASAP (1x effort)
>    - form to fill in: after that (3x effort)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/TeamMentor/TM_4_0_Design/issues/713#issuecomment-96221120>
> .
>



-- 
Michael Hidalgo
http://michaelhidalgocr.blogspot.com

*The future has many names: For the weak, it means the unattainable. For
the fearful, it means the unknown. For the courageous, it means
opportunity. (1802-1885) French Poet, Dramatist, Writer*

**Comment by roman87** [comment]
Ok, so we all agree its important, however for this release, and for simplicity, lets make this an email  link. 
@luchia We should change the copyright footer to say up on the very top of the footer - "Have comments on this article, send us an email..."  then the rest of the footer should follow. 

Everything else in this thread we should address as separate issues and in the next version :)

Also I am not sure we need terms and conditions link in the footer. Was that a business request original?

**Comment by luchia** [comment]
I think this looks good, what do you guys think?

![screen shot 2015-05-01 at 11 37 54 am](https://cloud.githubusercontent.com/assets/4547737/7425581/8e309b74-eff6-11e4-94df-2b320562c701.png)

**Comment by roman87** [comment]
I would make the text less encouraging in Jade. Simply because, as a user, when I see this, I expect to see a cool feedback form, with stars, and all that. So maybe more like "Have a comment about this article? Send our team an email"

**Comment by luchia** [comment]
Sure, i just fixed this on my branch and will create a PR shortly -

![screen shot 2015-05-05 at 5 12 08 am](https://cloud.githubusercontent.com/assets/4547737/7460312/5f839d7a-f2e5-11e4-8773-e58839085a61.png)

**Comment by luchia** [comment]
This has been fixed in my PR and will be merged into Dev shortly. I am marking as QA and assigning to Roman.

**Comment by roman87** [comment]
This is done.
